runtimes/alibi-explain/mlserver_alibi_explain/explainers/black_box_runtime.py

```diff
@@ -68,6 +67,10 @@ def _infer_impl(self, input_data: Union[np.ndarray, List[str]]) -> np.ndarray:
         # in the case of AnchorText, we have a list of strings instead though.
         # TODO: for now we only support v2 protocol, do we need more support?
 
+        if self.infer_metadata is None:
+            # get the metadata of the underlying inference model via v2 metadata endpoint
+            self.infer_metadata = remote_metadata(construct_metadata_url(self.infer_uri))
+
```

Comment:
`_infer_impl` is still going to be called via the `_load_from_uri` as alibi needs to make a dummy call at load time still.